src/components/UploadSection.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+    import {onMount} from 'svelte';
+    import { getUploadRequest } from '../lib/server/ApiCLient';
+  
+    let  files = [];
+    let uploadedFiles = [];
+
+    onMount(async()=>{
+
+        const get  = await getUploadRequest()
+        uploadedFiles = await get.json();
```

Comment:
This should look something like that:
```ts
const uploadRequests: UploadRequest[] = await fetchUploadRequests()
```
Also your fetch request can fail. So learn about exception handling.

Reply:
Thank you, make it with with type and handling exeption